packages/vendure-scripts/src/assign-these-prodcuts-to-channel.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  AssetService,
+  ChannelService,
+  FacetService,
+  ID,
+  Injector,
+  Product,
+  ProductVariantService,
+  RequestContext,
+  SearchService,
+  TransactionalConnection,
+} from '@vendure/core';
+import { getSuperadminContextInChannel } from '../../test/src/admin-utils';
+
+export async function assignTheseProductsToChannel(
```

Comment:
Can you add some JS doc here, stating that it commits all products in a single transaction, thus that it should be used with batches of products?

Reply:
I think is it best if we don't have batch level transactions. I will update the code to have the transaction started before all Products are processed and committed when we are done with importing all batches

Reply:
That makes sense :+1